src/hotspot/share/gc/g1/g1GCPhaseTimes.cpp

```diff
@@ -132,6 +132,8 @@ G1GCPhaseTimes::G1GCPhaseTimes(STWGCTimer* gc_timer, uint max_gc_threads) :
   _gc_par_phases[MergePSS]->create_thread_work_items("LAB Waste", MergePSSLABWasteBytes);
   _gc_par_phases[MergePSS]->create_thread_work_items("LAB Undo Waste", MergePSSLABUndoWasteBytes);
 
+  _gc_par_phases[RestoreRetainedRegions]->create_thread_work_items("Evacuation Failure Regions:", RestoreRetainedRegionsNum);
```

Comment:
Based on the surrounding naming conventions, `RestoreRetainedRegionsNum` should probably be sth like `RestoreRetainedRegionsNumEvacFailRegions` or just the suffix `NumEvacFailRegions`.

Reply:
Thanks Albert, I see your point.
But seems the surrounding naming convention is not that strict, and there is no uniform convention here. And to me, `RestoreRetainedRegionsNum` is more friendly to read and easily related to `RestoreRetainedRegions`, so I prefer to keep it as `RestoreRetainedRegionsNum`.